src/main/java/io/vlingo/actors/Stage.java

```diff
@@ -469,7 +452,11 @@ public World world() {
       final Actor actor = createRawActor(definition, parent, maybeAddress, maybeMailbox, maybeSupervisor, logger);
       final T protocolActor = actorProxyFor(protocol, actor, actor.lifeCycle.environment.mailbox);
       return new ActorProtocolActor<T>(actor, protocolActor);
-    } catch (Exception e) {
+    }
+    catch (Directory.ActorAddressAlreadyRegistered e) {
+      throw e;
+    }
+    catch (Exception e) {
```

Comment:
@VaughnVernon anything that you think could go wrong because of this? Tests say it is fine, but coverage is not that good.

Reply:
@buritos No problem that I see, but I think that this method must have the same:

`ActorProtocolActor<Object>[] actorProtocolFor(`

Reply:
Let's look into that when we do the work for making `lookupOrStart` and maybe friends part of the public API. Currently, `ActorProtocolActor<Object>[] actorProtocolFor(` is not in the way since there is no `lookupOrStart(Class<T>[] protocols ...)` variant.